OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/ViewKeyActivity.java

```diff
@@ -923,6 +924,7 @@ protected void onPostExecute(Bitmap photo) {
                                     }
 
                                     mPhoto.setImageBitmap(photo);
+                                    mPhoto.setColorFilter(Color.argb(30, 120, 166, 69));
```

Comment:
The color should be in resources, not hardcoded. Also, please explicitly set the blending mode :+1: